pkgs/applications/editors/neovim/utils.nix

```diff
@@ -176,7 +176,11 @@ let
         (lib.replaceStrings [ "-" ] [ "_" ])
       ];
 
-      nvimGrammars = lib.mapAttrsToList (name: value: value.origGrammar) vimPlugins.nvim-treesitter.grammarPlugins;
+      nvimGrammars = lib.mapAttrsToList (
+        name: value:
+        (value.origGrammar or builtins.throw
+          "additions to `pkgs.vimPlugins.nvim-treesitter.grammarPlugins` set should be passed through `pkgs.neovimUtils.grammarToPlugin` first")
+      ) vimPlugins.nvim-treesitter.grammarPlugins;
```

Comment:
```suggestion
      nvimGrammars = lib.mapAttrsToList (
        name: value:
        value.origGrammar
          or (builtins.throw "additions to `pkgs.vimPlugins.nvim-treesitter.grammarPlugins` set should be passed through `pkgs.neovimUtils.grammarToPlugin` first")
      ) vimPlugins.nvim-treesitter.grammarPlugins;
```

Reply:
thanks sorry about that

In my defense it works in the repl XD

```nix
nix-repl> test = { test2 = {}; }

nix-repl> tout = test.test1 or builtins.throw "message"

nix-repl> tout
error:
       … while calling the 'throw' builtin
         at «string»:1:2:
            1|  test.test1 or builtins.throw "message"
             |  ^

       error: message
```